borg/remote.py

```diff
@@ -414,6 +414,9 @@ class RepositoryCache(RepositoryNoCache):
 
     Caches Repository GET operations using a local temporary Repository.
     """
+    # maximum object size that will be cached, 64 kiB.
+    THRESHOLD = 2**16
```

Comment:
ok, so this caches just small files and item metadata chunks (always, as max chunk size for metadata is 2**16), but no file content data for files >2**16 length.

so, this will re-request content chunks from remote (even popular chunks like all-zero) all the time? hmm, if compression is on, that will save us at least for the all-x cases.